src/onnxruntime.cc

```diff
@@ -608,6 +608,51 @@ ModelState::LoadModel(
       }
     }
   }
+#ifdef TRITON_ENABLE_GPU
+  else {
```

Comment:
What about cases where the config provides the execution accelerators?  We are skipping adding those.

> I need to do some additional cleanup but as I understand the check at [https://github.com/triton-inference-server/onnxruntime_backend/blob/90c09650d190fd83f58fd4b1039acdc3092180fb/src/onnxruntime.cc#L391](https://nam11.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Ftriton-inference-server%2Fonnxruntime_backend%2Fblob%2F90c09650d190fd83f58fd4b1039acdc3092180fb%2Fsrc%2Fonnxruntime.cc%23L391&data=05%7C01%7Ctanmayv%40nvidia.com%7C94fbd1bddeeb4660d77808da223a6076%7C43083d15727340c1b7db39efd9ccc17a%7C0%7C0%7C637859931363782285%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=PEDfEkGdjFMDX6HJJEv62zE0%2FOVosC5PkMMGBfB4V0A%3D&reserved=0) is not needed and we can change L397 from (instance_group_kind == TRITONSERVER_INSTANCEGROUPKIND_GPU) to (instance_group_kind != TRITONSERVER_INSTANCEGROUPKIND_CPU)

Why are we not making the change you said we should here?

Reply:
Cleaned up. Please look at latest changes